Jenkinsfile

```diff
@@ -40,6 +41,19 @@ pipeline {
             }
         }
 
+		stage('test subsystem with podman') {
+				environment {
+					SERVICE = 'ocpmetal/assisted-service:test-onprem'
+				}
+			steps {
+				sh '''export PATH=$PATH:/usr/local/go/bin; make build-onprem'''
+				sh 'make deploy-onprem'
+				sleep 30
```

Comment:
What's this for?  Please add a comment, or preferably have the code wait on something rather than sleep for a fixed time.